pkg/common/providers/moaprovider/moa.go

```diff
@@ -35,3 +35,8 @@ func New() (*MOAProvider, error) {
 func (m *MOAProvider) Type() string {
 	return "moa"
 }
+
+// ExtendExpiry extends the expiration time of an existing cluster
+func (m *MOAProvider) ExtendExpiry(clusterID string, hours uint64, minutes uint64, seconds uint64) error {
+	return nil
```

Comment:
You can add this to the wrapped calls gofile and just call the OCM extend expiry function. This will be useful pretty quickly.